sd-ran/values.yaml

```diff
@@ -0,0 +1,144 @@
+# Default values for all SD-RAN Helm charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+import:
+  nem-monitoring:
+    enabled: true
+  onos-ric:
+    enabled: true
+  onos-ric-ho:
+    enabled: true
+  onos-ric-mlb:
+    enabled: true
+  onos-topo:
+    enabled: true
+  onos-gui:
+    enabled: true
+  ran-simulator:
+    enabled: true
+
+# NEM-Monitoring
+nem-monitoring:
+  prometheus:
+    serverFiles:
+      prometheus.yml:
+        scrape_configs:
+          - job_name: 'onos-ric'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - onos-ric.micro-onos.svc.cluster.local:7000
+          - job_name: 'onos-ric-ho'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - onos-ric-ho.micro-onos.svc.cluster.local:7001
+          - job_name: 'onos-ric-mlb'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - onos-ric-mlb.micro-onos.svc.cluster.local:7002
+          - job_name: 'ran-simulator'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - ran-simulator.micro-onos.svc.cluster.local:9090
+  dashboards:
+    nodeExporter: false
+  kpi_exporter: 
+    enabled: false
+  grafana:
+    datasources:
+      datasources.yaml:
+        apiVersion: 1
+        datasources:
+          - name: Prometheus
+            type: prometheus
+            url: http://sd-ran-prometheus-server
+            access: proxy
+            isDefault: true
+
+# ONOS-RIC
+onos-ric:
+  image:
+    repository: onosproject/onos-ric
+    tag: latest
+    pullPolicy: IfNotPresent
+  debug: false
+  store:
+    enabled: true
```

Comment:
I think the store configurations need to be updated after we merged #51 
Also, for the umbrella chart, we may want to disable stores for the individual services and deploy a single consensus `Database` and configure each service to use the same database.

Reply:
Oh,ok. Let me change here.

Reply:
Updated.